crates/bevy_reflect/src/func/into_function.rs

```diff
@@ -118,12 +118,12 @@ macro_rules! impl_into_function {
             R: $crate::func::IntoReturn + $crate::func::args::GetOwnership + $crate::TypePath,
             F: FnMut($($Arg),*) -> R + 'env,
             F: for<'a> FnMut($($Arg::Item<'a>),*) -> R + 'env,
+            F: Send + Sync,
```

Comment:
Oh, I didn't know you could add multiple bounds on the same thing like this.

Reply:
I learned this recently too (I think while working on function reflection)! It definitely helps break them up for readability imo